tests/migration/Core/V6_5/Migration1663402950SetDoubleOptinCustomerActiveTest.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_5;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Migration\V6_5\Migration1663402950SetDoubleOptinCustomerActive;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Migration\V6_5\Migration1663402950SetDoubleOptinCustomerActive
+ */
+class Migration1663402950SetDoubleOptinCustomerActiveTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    private Connection $connection;
+
+    public function setUp(): void
+    {
+        $this->connection = KernelLifecycleManager::getConnection();
+    }
+
+    public function testMigration(): void
+    {
+        $migration = new Migration1663402950SetDoubleOptinCustomerActive();
+
+        $customerId = Uuid::randomBytes();
+        $customerAddressId = Uuid::randomBytes();
+        $now = new \DateTimeImmutable();
+
+        $countAffectedRows = $this->connection->insert('customer', [
+            'id' => $customerId,
+            'customer_group_id' => Uuid::fromHexToBytes(Defaults::FALLBACK_CUSTOMER_GROUP),
+            'default_payment_method_id' => Uuid::fromHexToBytes($this->getValidPaymentMethodId()),
```

Comment:
```suggestion
            'default_payment_method_id' => $connection->fetchOne('SELECT id FROM `payment_method` WHERE `active` = 1'),
```
you can use a inline query